### PR TITLE
AVX-15478: Add advanced options for BGP spoke gateway

### PR DESCRIPTION
**Pull request by zhiqiangjin**

### CODE REVIEW

File: aviatrix/resource_aviatrix_spoke_gateway.go [hunk 1]
```diff
@@ -2122,6 +2211,65 @@ func resourceAviatrixSpokeGatewayUpdate(d *schema.ResourceData, meta interface{}
...
+		err := client.SetBgpPollingTimeSpoke(gateway, bgpPollingTime)
+		if err != nil {
+			return fmt.Errorf("could not update bgp polling time: %v", err)
+		}
+	}
+
+	if d.HasChange("bgp_hold_time") {
+		err := client.ChangeBgpHoldTime(gateway.GwName, d.Get("bgp_hold_time").(int))
+		if err != nil {
+			return fmt.Errorf("could not change BGP Hold Time during Transit Gateway update: %v", err)
```
Comment (mlin-aviatrix) [inline aviatrix/resource_aviatrix_spoke_gateway.go]: Should be Spoke Gateway not Transit Gateway

File: aviatrix/resource_aviatrix_spoke_gateway.go [hunk 2]
```diff
@@ -294,6 +295,40 @@ func resourceAviatrixSpokeGateway() *schema.Resource {
...
+			},
+			"enable_active_standby": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     false,
+				Description: "Enables Active-Standby Mode, available only with HA enabled.",
+			},
+			"prepend_as_path": {
+				Type:        schema.TypeList,
+				Optional:    true,
```
Comment (shuoliugit) [inline aviatrix/resource_aviatrix_spoke_gateway.go:319]: If this is a list, is it possible that backend return something in a different order, causing a diff?

Comment (mlin-aviatrix) [reply aviatrix/resource_aviatrix_spoke_gateway.go:319]: prepend_as_path is a list of repeated as num for the gateway so order shouldn't matter

Comment (zhiqiangjin) [reply aviatrix/resource_aviatrix_spoke_gateway.go:319]: Correct, it does not matter.

**Review by mlin-aviatrix — DISMISSED** [review]

**Review by shuoliugit — APPROVED** [review]

**Review by mlin-aviatrix — APPROVED** [review]

**Pull request merged**